activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java

```diff
@@ -337,14 +331,7 @@ public ProcessInstance update(UpdateProcessPayload updateProcessPayload) {
                                                                              .processInstanceId(updateProcessPayload.getProcessInstanceId())
                                                                              .singleResult());
         
-        fireProcessUpdatedEvent(updatedProcessInstance);
         return updatedProcessInstance;
  
     }
-    
-    private void fireProcessUpdatedEvent(ProcessInstance updatedProcessInstance) {
```

Comment:
@igdianov I don't see a `registerProcessUpdatedEventListener` replacing this.

Reply:
Ignore this comment, the registration is already taking place in develop https://github.com/Activiti/Activiti/blob/develop/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/conf/activiti/runtime/api/ProcessRuntimeAutoConfiguration.java#L185